src/clojure/clj_arangodb/arangodb/oasis.clj

```diff
@@ -0,0 +1,22 @@
+(ns clj-arangodb.arangodb.oasis
+  (:import
+   [java.io ByteArrayInputStream]
+   [javax.net.ssl SSLContext TrustManagerFactory]
+   [java.security KeyStore]
+   [java.security.cert CertificateException CertificateFactory X509Certificate]
+   [java.util Base64]))
+
+;; ====  SSL  ==================================================================
+
+(defn initialize-ssl-context
+  [^String certificate]
+  (let [is (new ByteArrayInputStream (.decode (Base64/getDecoder) certificate))
+        cf (CertificateFactory/getInstance "X.509")
+        caCert (.generateCertificate cf is)
+        tmf (TrustManagerFactory/getInstance (TrustManagerFactory/getDefaultAlgorithm))
+        ks (KeyStore/getInstance (KeyStore/getDefaultType))
+        ssl-context (SSLContext/getInstance "TLS")]
+    (.load ks nil)
+    (.setCertificateEntry ks "caCert" caCert)
+    (.init tmf ks)
+    (.init ssl-context nil (.getTrustManagers tmf) nil)))
```

Comment:
It seems like `.init` returns `null` as per https://docs.oracle.com/javase/7/docs/api/javax/net/ssl/SSLContext.html
If so then this function returns `nil` which is probably not what you want. 

I would assume that you should be returning `ssl-context` at the end of this function so that the arangodb `builder` gets an object. 

https://github.com/arangodb/arangodb-java-driver/blob/master/src/test/java/com/arangodb/example/ssl/SslExample.java#L63

https://github.com/arangodb/arangodb-java-driver/blob/d0fcc9c53686026c41035f41c10f043c63a0ecb6/src/test/java/com/arangodb/example/ssl/SslExample.java#L87